tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py

```diff
@@ -133,19 +133,16 @@ def parse_args():
     logger = logging.getLogger("submit_groupwise_azureml_pytest.py")
     args = parse_args()
 
-    logger.info(f"Setting up workspace {args.ws}")
+    logger.info("Setting up workspace %s", args.ws)
```

Comment:
I think we should roll back to using f-strings. It is faster than the % operator. Also, in other parts of the code we use f-strings. 

What do you think?

Reply:
This change is suggested by pylint: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html
But I haven't go deeper to understand why.

Reply:
I guess using f-strings makes "python" do the conversion to string, while using the logger arguments allows the logging library to call __repr__ instead, do some kind of introspection, or even handle the error in some special way.

Reply:
Oh, and its also "lazier". It is only converted to string if that level of logging is enabled

Reply:
Yeah it looks like a small time saving according to https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation